### ISSUE TITLE
`SetTitle` is not working

### ISSUE THREAD

**Issue by upendra-web** [fyne-io/systray#26]
Hi, thanks for creating a fork of [lantern's systray](https://github.com/getlantern/systray).

I tried the below code with systray library
```go
package main

import (
	"fyne.io/systray"
	"fyne.io/systray/example/icon"
)

func main() {
	systray.Run(onReady, onExit)
}

func onReady() {
	systray.SetIcon(icon.Data)
	systray.SetTitle("Awesome App")
	systray.SetTooltip("Pretty awesome超级棒")
	mQuit := systray.AddMenuItem("Quit", "Quit the whole app")

	// Sets the icon of a menu item. Only available on Mac and Windows.
	mQuit.SetIcon(icon.Data)
}

func onExit() {
	// clean up here
}
```

But I couldn't see the title in system tray
![systray_issue](https://user-images.githubusercontent.com/43108965/194757791-3601aab2-a0ca-4770-b013-f3a9da8fcec9.png)

So was this intentional? Or this is an issue? Thanks in advance!

### System and package details

**OS**: Ubuntu 21.10
**Desktop environment**: Gnome 40.4.0
**Package version**: fyne.io/systray v1.10.0
**Go version**: 1.18

**Comment by andydotxyz** [comment]
Can you confirm where the title was expected to appear? My understanding is that it appears on tooltip, but it was not clear from the screenshot that this was not the case?

**Comment by upendra-web** [comment]
I tried the below code with `latern's systray` module
```go
package main

import (
	"github.com/getlantern/systray"
	"github.com/getlantern/systray/example/icon"
)

func main() {
	systray.Run(onReady, onExit)
}

func onReady() {
	systray.SetIcon(icon.Data)
	systray.SetTitle("Awesome App")
	systray.SetTooltip("Pretty awesome超级棒")
	mQuit := systray.AddMenuItem("Quit", "Quit the whole app")

	// Sets the icon of a menu item. Only available on Mac and Windows.
	mQuit.SetIcon(icon.Data)
}

func onExit() {
	// clean up here
}
```
It rendered `the title beside the icon` as below
![systray_latern](https://user-images.githubusercontent.com/43108965/194805112-aff04957-8941-4672-b8e6-8995b96ff89a.png)

So with fyne.io's systray module I thought the result might be the same.

**Comment by upendra-web** [comment]
And just FYI, the `Quit` you see in the screenshot is a menu option, not a tooltip.

**Comment by upendra-web** [comment]
Hi @andydotxyz, any updates on the above, is it an issue or design choice?

**Comment by andydotxyz** [comment]
I have checked out the protocol and the code, it should work.
Do you see any error on the command line when running the app?

**Comment by andydotxyz** [comment]
It could be a difference in how your window manager handles legacy system tray and the new "AppIndicator" tray items, i'm not entirely sure, as Gnome does not work on this computer.
We are passing the title to the DE correctly as far as I can see, but it is up to the panel whether or not it is displayed.
However in the old model the app controls what is drawn, so the `getlantern` code may have been forcing it before?

**Comment by upendra-web** [comment]
Yeah, that might be the problem of the DE, I will also try to check the `getlatern`'s code. Thanks @andydotxyz for the clarification. 

Can you clarify a doubt, is it ok to encode a SVG(which has some text) to byte array and pass that to `SetIcon`?

**Comment by andydotxyz** [comment]
> Can you clarify a doubt, is it ok to encode a SVG(which has some text) to byte array and pass that to `SetIcon`?

I'm pretty sure that the image data must be bitmap (.png even).
In the Fyne library we can rasterise SVG, but the OS systray cannot and this library does not have that feature either.

**Comment by henri9813** [comment]
Hello,

Any update on this subject? why it works on get lantern initial project and not this one ?

On MacOS, I don't have the issue but I got it on windows.

**Comment by andydotxyz** [comment]
Windows does not support having text next to the system tray icon, so it will show as a tooltip (if none other is set) instead.

**Comment by henri9813** [comment]
Thanks for your feedback.

Maybe could we add this to documentation ?

**Comment by ar065** [comment]
Having the same issue here on KDE.

**Comment by andydotxyz** [comment]
I don't think KDE supports inserting text to the system tray. Can you show any app which does that?

**Comment by andydotxyz** [comment]
I don't think there is anything remaining to do on this
